### DISCUSSION TITLE
I don't know how to send a WEBHOOK message to a URL

### DISCUSSION THREAD

**Discussion by ED054** [community/community#106912] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey.
I am new.
I am trying to create a workflow.
I succeeded to connect an SMS to an alternate email but I don't know how to continue from here.
I have a url to send a webhook message to.
This is the address: https://tvwebhook.capitalise.ai
This is the webhook message: {"alertId": "d3bb481b-7bd4-4a15-bd0c-15dfc371ef8a"}

I don't know what the next step is to do :(
I don't know what to choose after the + and what I should set.
That's all I need to just send this message to this address.

this is my workflow: https://pipedream.com/@ed0545692461/projects/proj_zNsq9GA/untitled-workflow-7-2-2024-16-27-p_rvCmqOm/build

**Comment by ZephyrusBlaze** [comment]


**Reply by ED054** [reply]
There are many HTTP request options and I don't know what to choose.
And I don't know what GET and POST are either.
And where exactly to paste the message

**Reply by ZephyrusBlaze** [reply]
- In the URL field, paste `https://tvwebhook.capitalise.ai`.
- Scroll down or look for a section labeled "Body" or "Request Body".
-  Paste `{"alertId": "d3bb481b-7bd4-4a15-bd0c-15dfc371ef8a}` into this field.
- For now, you can choose "POST" from the dropdown menu (it's a common choice for sending data).

**Reply by ED054** [reply]
ok thank you for your help
But it gave me an error message.

TIMEOUT: TIMEOUT
     at Timeout._onTimeout (file:///var/task/lambda_handler.mjs:934:23)
     at listOnTimeout (node:internal/timers:573:17)
     at process.processTimers (node:internal/timers:514:7)

**Reply by ED054** [reply]
I have to go.
Thank you, I will try again later today
